teku/src/main/java/tech/pegasys/teku/cli/options/ExecutionEngineOptions.java

```diff
@@ -36,8 +36,20 @@
       hidden = true)
   private Version executionEngineVersion = Version.DEFAULT_VERSION;
 
+  @Option(
+      hidden = true,
+      names = {"--Xjwt-secret"},
```

Comment:
For consistency we probably should call this:
```suggestion
      names = {"--Xee-jwt-secret"},
```